index.html

```diff
@@ -0,0 +1,89 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="the author Portfolio" />
+    <meta name="author" content="Arthur Emanuel Guerra Iturres" />
+    <link
+      rel="icon"
+      href="src/assets/media/tab-app-icons/illustration-tab-icon.png"
+    />
+    <link
+      rel="stylesheet"
+      href="src/assets/styles/main-stylesheets/index.css"
+    />
+    <title>Arthur.Iturres | Portfolio</title>
+  </head>
+  <body id="go-top">
+    <nav id="my-navbar" class="my-navbar">
+      <ul class="hidden">
+        <li>
+          <a class="my-navbar__link" href="#go-top">Portfolio</a>
+        </li>
+        <li><a class="my-navbar__link" href="#about-section">About</a></li>
+        <li><a class="my-navbar__link" href="#contact-section">Contact</a></li>
+      </ul>
+      <span class="close-my-navbar__btn my-btn hidden" data-open-my-navbar>
+        <i class="fa-solid fa-x"></i>
+      </span>
+      <div class="open-my-navbar">
+        <span class="open-my-navbar__btn my-btn" data-close-my-navbar>
+          <i class="fa-solid fa-bars"></i>
+        </span>
+      </div>
+    </nav>
+    <header class="my-header">
+      <div class="my-header__title--wrapper">
+        <h1 class="my-header__title">Hey there.</h1>
+        <h2 class="my-header__title">I'm Arthur.</h2>
+      </div>
+      <h3 class="my-header__subtitle">I'm a software developer</h3>
+      <p class="my-header__intro">
+        I can help you build a product, feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don't hesitate to contact me.
+      </p>
+      <nav class="my-header__nav">
+        <ul>
+          <li>
+            <a href="#" class="my-header__link"
+              ><img
+                src="src/assets/media/sections-illustrations/social-icons/twitter.png"
+                alt=""
+                class="link__icon" /></a
+            ><a href="#" class="my-header__link"
+              ><img
+                src="src/assets/media/sections-illustrations/social-icons/linkedin.png"
+                alt=""
+                class="link__icon" /></a
+            ><a href="#" class="my-header__link"
+              ><img
+                src="src/assets/media/sections-illustrations/social-icons/medium.png"
+                alt=""
+                class="link__icon" /></a
+            ><a href="#" class="my-header__link"
+              ><img
+                src="src/assets/media/sections-illustrations/social-icons/github.png"
+                alt=""
+                class="link__icon" /></a
+            ><a href="#" class="my-header__link"
+              ><img
+                src="src/assets/media/sections-illustrations/social-icons/angellist.png"
+                alt=""
+                class="link__icon"
+            /></a>
+          </li>
```

Comment:
- [ Required ] I think it would be better if each `a` tag is wrapped in its own `li` tag, currently, all the `a` tags use a single `li` tag which is not semantically correct. 

- [ Required ] It would also be better if the `href` attributes to your social media pages are not empty and had links that lead to your social media pages in case a recruiter wants to reach out to you. 

## Example code
```html
  <ul>
   <li>
      <a href="https://twitter.com/" class="my-header__link">
         <img  src="src/assets/media/sections-illustrations/social-icons/twitter.png" 
                   alt="" class="link__icon" /></a>
  </li>
  <li>
      <a href="https://www.linkedin.com/" class="my-header__link">
         <img src="src/assets/media/sections-illustrations/social-icons/linkedin.png"  
                  alt="" class="link__icon" />
     </a>
    </li>
  </ul>
```

Reply:
silly me!, you are right, I placed all the anchors within a single <li> tag 🤦‍♂️. and thank you again because yes, I forgot to set the href attributes. will fix it right away!